scripts/run_dev.sh

```diff
@@ -28,7 +28,7 @@ ISAAC_ROS_DEV_DIR="$1"
 if [[ -z "$ISAAC_ROS_DEV_DIR" ]]; then
     ISAAC_ROS_DEV_DIR="$HOME/workspaces/isaac_ros-dev"
     if [[ ! -d "$ISAAC_ROS_DEV_DIR" ]]; then
-        ISAAC_ROS_DEV_DIR=$(pwd)
+        ISAAC_ROS_DEV_DIR="$ROOT/../../.."
```

Comment:
Assuming the relative path to ISAAC_ROS_DEV_DIR is not guaranteed here. The intent was to recognize that the user had not chosen to use the same directory hierarchy as we do, so take their current PWD as their dev dir. I see your point though that current dir assumes you're running `<MY_ISAAC_ROS_DEV_DIR?$ isaac_ros_common/scripts/run_dev.sh` which is not guaranteed. Moving up two directories should be enough to get you into the root directory containing isaac_ros_common which is sufficient if you agree.

Reply:
The fix provided here is not for arbitrary directory hierarchy (e.g., without the `src` directory). Instead, it is for arbitrary `ISAAC_ROS_DEV_DIR` path. Specifically, the [setup tutorial](https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_common/blob/main/docs/dev-env-setup.md) uses different path for x86/ARM CPUs:
- x86: `~/workspaces/isaac_ros-dev/src/isaac_ros_common/scripts`
- ARM: `/ssd/workspaces/isaac_ros-dev/src/isaac_ros_common/scripts`

The script prior to this fix works on the first (x86) path, but not on the second (ARM) path.

Given that users will follow the official tutorial, we can assume the relative path of `isaac_ros-dev` and `isaac_ros_common` is `isaac_ros-dev/src/isaac_ros_common`, which requires moving up three directories.

If we only move up two directories, the mounted volume will be incorrect. Resulting in mounting `<SOME_PATH>/isaac_ros-dev/src` to `/workspaces/isaac_ros-dev`, which will make all commands executed in the container fail when following the tutorial. See below:

https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_common/blob/88023df7376ac02421302a61349e439165557778/scripts/run_dev.sh#L183